src/utils/mmap_backend.rs

```diff
@@ -181,6 +196,11 @@ impl<W> MmapBackend<W, MmapMut> {
                 format!("Cannot create {} new MmapBackend", path.as_ref().display())
             })?;
         let file_len = len * size_of::<W>();
+        #[cfg(windows)]
```

Comment:
Didn't we say we don't write to files?

Reply:
Same as above

Reply:
What about this?
Whether we truncate the file or simply create it, it will always have size 0.
If we try to open a non-zero sized memory map, Windows will fill it with random bytes.
If we fail if the file size is different than the memory map, this leads to a function that always fails

Reply:
Mmmhhh... this is not very clear; please try to explain better. Why truncate?